lib/backend/syncersv1/bgpsyncer/bgpsyncer.go

```diff
@@ -1,67 +0,0 @@
-// Copyright (c) 2017-2018 Tigera, Inc. All rights reserved.
```

Comment:
Ah, I guess because we're moving this logic into the confd codebase? 

Here - https://github.com/projectcalico/confd/pull/145/files#diff-0fe69fc83f76166f830461b1ef30963bL111

Curious, I would have thought this would make our CI fail. hmmmm.

Reply:
Yep - about moving into confd.  What CI do you think would fail?

Reply:
I would have thought this part would fail: https://github.com/projectcalico/libcalico-go/blob/master/Makefile#L255-L264

We have a target which builds libcalico users - I'd expect building to fail when we remove the BGP syncer code.